src/control/cmd/daos_agent/config_test.go

```diff
@@ -175,3 +178,28 @@ transport_config:
 		})
 	}
 }
+
+/*func TestAgent_Config_CacheRefreshInterval(t *testing.T) {
```

Comment:
should this be removed or uncommented?

Reply:
Ah, good catch, I forgot to remove it since it was no longer necessary.